### PR TITLE
Fixes #14126 - support a max height and scroll on autocomplete

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: app/assets/stylesheets/bastion/nutupane.scss [hunk 1]
```diff
@@ -398,3 +398,8 @@ div.alch-dialog.open.info-value {
 table .progress {
   margin-bottom: 0;
 }
+
+.dropdown-menu {
+    max-height: 500px;
```
Comment (waldenraines) [inline app/assets/stylesheets/bastion/nutupane.scss]: On the foreman side they use 300px, perhaps we should use 300px as well?

**Comment by jlsherrill** [comment]
updated to 300px

**Comment by waldenraines** [comment]
ACK

**Comment by jlsherrill** [comment]
@waldenraines added overflow-x: hidden;  as the bookmarks drop down had a horizontal scroll

**Comment by waldenraines** [comment]
> @waldenraines added overflow-x: hidden; as the bookmarks drop down had a horizontal scroll
>  Show all checks

What happens if the suggestion is longer than the window?  Do we need to add some word wrapping?

**Comment by jlsherrill** [comment]
@waldenraines the foreman auto complete auto expands horizontally rather than allowing horizontal scrolling.  We should probably end up doing that.

**Comment by jlsherrill** [comment]
@waldenraines updated to add auto-expanding

**Comment by waldenraines** [comment]
ACK

File: app/assets/stylesheets/bastion/nutupane.scss [hunk 2]
```diff
@@ -398,3 +398,9 @@ div.alch-dialog.open.info-value {
 table .progress {
   margin-bottom: 0;
 }
+
+.dropdown-menu {
+    max-height: 300px;
+    overflow-y: auto;
+    overflow-x: hidden;
```
Comment (waldenraines) [inline app/assets/stylesheets/bastion/nutupane.scss]: Actually never mind, it is needed.

**Comment by waldenraines** [comment]
ACK

**Pull request merged**